File: JavaScript/bee 1176.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

function fibonacciVetor(quant){
    let sequenciaFibonacci = [0,1]
    let soma = 0
    var num1 = 0
    var num2 = 0
    for(var i = 1; i <= quant - 1; i++){
        num1 = sequenciaFibonacci[i - 1]
        num2 = sequenciaFibonacci[i]
        soma = num1 + num2
        sequenciaFibonacci.push(soma)
    }
    return sequenciaFibonacci
}


var n = Number(lines.shift())
for(var i = 0; i < n; i++){
    var num = Number(lines[i])
    sequenciaFibonacci = fibonacciVetor(num)
    console.log(`Fib(${num}) = ${sequenciaFibonacci[num]}`)
}

File: Java/bee 1176.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        for(int i = 0; i < t; i++) {
            int n = sc.nextInt();
            System.out.printf("Fib(%d) = %d\n", n, fib(n));
        }
    }
    public static long fib(int n) {
        if(n == 0 || n == 1) {
            return n;
        }

        long soma = 0;
        long a = 0;
        long b = 1;
        for(int i = 2; i <= n; i++) {
            soma = a + b;
            a = b;
            b = soma;
        }

        return soma;
    }
}
